app/retail/templates/mission.html

```diff
@@ -144,7 +150,9 @@ <h4 class="text-left text-uppercase">
 
           <div class="divider"></div>
 
+          {% blocktrans %}
           <p>Intelectual honesty</p>
```

Comment:
Can we fix the spelling here to `Intellectual` - Sorry, missed this the first time!

Reply:
fixed